docs/source/config_options.rst

```diff
@@ -1,194 +1,222 @@
-| brian_to_radctrl_identity               | BRIAN_RADCTRL_IDEN      | ZMQ named socket identity.           |
-+-----------------------------------------+-------------------------+--------------------------------------+
-| brian_to_driver_identity                | BRIAN_DRIVER_IDEN       | ZMQ named socket identity.           |
-+-----------------------------------------+-------------------------+--------------------------------------+
-| brian_to_dspbegin_identity              | BRIAN_DSPBEGIN_IDEN     | ZMQ named socket identity.           |
-+-----------------------------------------+-------------------------+--------------------------------------+
-| brian_to_dspend_identity                | BRIAN_DSPEND_IDEN       | ZMQ named socket identity.           |
-+-----------------------------------------+-------------------------+--------------------------------------+
-| ringbuffer_name                         | data_ringbuffer         | Shared memory name for ringbuffer.   |
-+-----------------------------------------+-------------------------+--------------------------------------+
-| ringbuffer_size_bytes                   | 200.00E+06              | Size in bytes to allocate for each   |
-|                                         |                         | ringbuffer.                          |
-+-----------------------------------------+-------------------------+--------------------------------------+
-| data_directory                          | /data/borealis_data     | Location of output data files.       |
-+-----------------------------------------+-------------------------+--------------------------------------+
++--------------------------------+-------------------------------+---------------------------------------+
+| Config field                   | Example entry                 | Description                           |
++================================+===============================+=======================================+
+| site_id                        | sas                           | 3-letter standard ID of the radar     |
++--------------------------------+-------------------------------+---------------------------------------+
+| gps_octoclock_addr             | addr=192.168.10.131           | IP address of the GPS Octoclock       |
++--------------------------------+-------------------------------+---------------------------------------+
+| device_options                 | recv_frame_size=4000          | UHD USRP device arguments.            |
++--------------------------------+-------------------------------+---------------------------------------+
+| main_antennas                  | 0,1,2,3,4,5,6,7,8,9,          | Mapping of main rx/tx channels to     |
+|                                | 10,11,12,13,14,15             | antennas in the main array            |
++--------------------------------+-------------------------------+---------------------------------------+
```

Comment:
I think these two fields can go now, right? main_antennas and interferometer_antennas? I know these are included in the main branch docs, but I think they are redundant here now with these changes.

Reply:
I believe they can go also, I just have to add parsing code to create these fields within the various options files to match the code in main. 

It looks like in main, `main_antenna_count` just refers to the number of antennas in the physical array, while in this PR I have it referring to the total number of transmitting/receiving antennas. Should I modify this PR to be more similar to main? Or leave that for merge conflict resolution?

Reply:
Is it the total number of tx/rx antennas, or the range of the indices? If you are using only antennas [0, 1, 4, 6], will `main_antenna_count` report 4 or 7? 

Honestly I'm not sure, I think we're going to have a heck of a time with the merge conflicts when merging develop into main regardless of what is done here. It might be better just to kick this can down the road rather than relive the same sort of headache twice in trying to figure it out now plus again when we merge that PR.

Reply:
Nevermind, I think I have a solid opinion on this. I think it would be best to keep `main_antenna_count` and `interferometer_antenna_count` as fields in the config file SEPARATE from the N200 fields. They will be fields that are basically only used to check the validity of the antenna indices within the N200 fields. 

I was looking at how you build up the `..._antenna_count` fields now: [Line 44](https://github.com/SuperDARNCanada/borealis/pull/334/files#diff-148a6e57c058ff1bde9db54347b543dcbf4889310b8ff83240fca83df35363c1R44)
and I think that this can lead to errors down the road if someone makes a mistake in their config file, based on a future check you make: [Line 101](https://github.com/SuperDARNCanada/borealis/pull/334/files#diff-148a6e57c058ff1bde9db54347b543dcbf4889310b8ff83240fca83df35363c1R101). So I would recommend we make the `..._antenna_count` field a constant as I explained in the previous paragraph.